ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/TESTS_PLATFORM.yml

```diff
@@ -0,0 +1,20 @@
+type: l2leaf
+
+l2leaf:
+  node_groups:
+    TESTS_PLATFORM_7368X4:
+      mlag: true
+      mgmt_gateway: 42.42.42.1
+      platform: 7368X4
+      uplink_interfaces:
+      mlag_interfaces: [ Ethernet3 ]
+      mlag_peer_vlan: 4091
+      mlag_peer_ipv4_pool: 192.168.255.0/24
+      rack:
```

Comment:
Any reason to keep this blank. Please remove this if not required.

Reply:
Removed it.